test/libs/datasets/timeseries_test.py

```diff
@@ -29,6 +29,12 @@
 pytestmark = pytest.mark.filterwarnings("error", "ignore::libs.pipeline.BadFipsWarning")
 
 
+DEFAULT_FIPS = "97222"
```

Comment:
Super small, but I prefer naming this the name of the region this represents.  Not sure why i would expect this fips to be the default (or even where it is)